Sources/GDSAnalytics/Screens/ScreenView.swift

```diff
@@ -1,23 +1,25 @@
 public protocol ScreenViewProtocol {
     associatedtype Screen: NamedScreen
     var screen: Screen { get }
-    var title: String { get }
     var parameters: [String: String] { get }
 }
 
 public struct ScreenView<Screen: NamedScreen>: ScreenViewProtocol {
     public let screen: Screen
     public let title: String
+    public let screenID: String
```

Comment:
please can we make this optional to begin with so we don't break our backward compatibility?

Reply:
I thought the idea of having the packages versioned meant that we dont have to worry as much about backwards compatibility especially given that we are adding mandatory changes - making this optional means it could be missed when apps update to the newest schema.

Reply:
Versioning allows us to introduce breaking changes if we absolutely need to. Ideally I would rather avoid them completely if we can; we don't want a team getting too far behind and we need to give a considerable grace period to update.

Making changes optional also means teams can upgrade screen by screen rather than needing to do a big bang migration. Once we're confident all teams are up-to-date, then yes, we should make it a required field so that we are sure it doesn't get missed! 😄